autoload/airline/themes/dracula.vim

```diff
@@ -14,14 +14,10 @@
 " more attr-list items as input, transforms it to the format accepted by
 " airline#themes#generate_color_map and returns that value
 func! s:clr(fg, bg, ...)
-  let l:fg = get(g:dracula_palette, a:fg)
-  let l:bg = get(g:dracula_palette, a:bg)
-  if ! type(l:fg) == 3
-    echoerr 'Color undefined in dracula_palette: ' . a:fg
-  elseif ! type(l:bg) == 3
-    echoerr 'Color undefined in dracula_palette: ' . a:fg
-  endif
-  return [ l:fg[0], l:bg[0], l:fg[1], l:bg[1] ] + filter(copy(a:000), 'type(v:val) == 1 && len(v:val) > 0')
+  let l:fg = g:dracula#palette[a:fg]
+  let l:bg = g:dracula#palette[a:bg]
+  return [ l:fg[0], l:bg[0], l:fg[1], l:bg[1] ] +
+       \ filter(copy(a:000), {_, val -> type(val) == 1 && len(val) > 0 })
```

Comment:
Been on vacation for a bit, so just seeing this now. Everything looks good, although I can see this lambda syntax being an issue for older vims (which don't support it).

I don't *mind* not supporting older vims, but we should be explicit about it (and announce some kind of breaking change/increment a version number/whatever we have to do). We may want to solicit feedback about it.

Supporting older vims is as simple as converting the lambda back to a string using `v:val`.

Reply:
We can revert that line. Not worth breaking over something so trivial.